auth0/structure_auth0_connection.go

```diff
@@ -18,8 +18,8 @@ func flattenConnectionOptions(d Data, options interface{}) []interface{} {
 		m = flattenConnectionOptionsGoogleOAuth2(o)
 	// case *management.ConnectionOptionsFacebook:
 	// 	m = flattenConnectionOptionsFacebook(o)
-	// case *management.ConnectionOptionsApple:
-	// 	m = flattenConnectionOptionsApple(o)
```

Comment:
Could you remove these please?

Reply:
Not sure what you mean here. 😅

You want me to put the commented out code back in?

Reply:
Oh, my bad! You did remove them, I somehow didn’t notice the huge red highlighting over the line... 🙈😂